CNV/cnv-upgrade.sh

```diff
@@ -2,20 +2,14 @@
 
 set -ex
 
-OLD_CNV_VERSION="${OLD_CNV_VERSION:-2.0.0}"
-CNV_VERSION="${CNV_VERSION:-2.1.0}"
-OLD_CNV_CHANNEL="${OLD_CNV_VERSION:0:3}"
-CNV_CHANNEL="${CNV_VERSION:0:3}"
 TARGET_NAMESPACE="${TARGET_NAMESPACE:-openshift-cnv}"
 
-oc get sub hco-subscription -o yaml -n "${TARGET_NAMESPACE}" | sed "s/channel: ${OLD_CNV_CHANNEL}/channel: ${CNV_CHANNEL}/" | oc apply -n "${TARGET_NAMESPACE}" -f -
+oc get installplan -o yaml -n ${TARGET_NAMESPACE} $(oc get installplan -n ${TARGET_NAMESPACE} --no-headers | grep kubevirt-hyperconverged-operator.v2.1.0 | awk '{print $1}') | sed 's/approved: false/approved: true/' | oc apply -n openshift-cnv -f -
```

Comment:
z-stream upgrade is a different procedure than y-stream (2.1 -> 2.2).  I think we should have separate scripts to test each type of upgrade.  What do you think about putting this in `z-stream-upgrade.sh`?